<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressBarTest</class>
 <widget class="QWidget" name="ProgressBarTest">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>737</width>
    <height>454</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>多态进度条</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>115</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="TaiChiWait" name="taichiWait" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="CircleWait" name="circleWait" native="true"/>
      </item>
      <item row="0" column="2">
       <widget class="PieWait" name="pieWait" native="true"/>
      </item>
      <item row="0" column="3">
       <widget class="LineWait" name="lineWait" native="true"/>
      </item>
      <item row="0" column="4">
       <widget class="DonutWait" name="donutWait" native="true"/>
      </item>
      <item row="0" column="5">
       <widget class="QWidget" name="widget_3" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="ZoomCircleWait" name="zoomCircleWait1" native="true"/>
         </item>
         <item>
          <widget class="ZoomCircleWait" name="zoomCircleWait2" native="true"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>圆环风格</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>饼状风格</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>线条风格</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="RoundProgressBar" name="roundBar1" native="true">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Shadow">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>156</red>
                <green>220</green>
                <blue>207</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>24</red>
                <green>189</green>
                <blue>155</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Shadow">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>156</red>
                <green>220</green>
                <blue>207</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>24</red>
                <green>189</green>
                <blue>155</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Shadow">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>156</red>
                <green>220</green>
                <blue>207</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>51</red>
                <green>153</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="RoundProgressBar" name="roundBar2" native="true">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>97</red>
                <green>117</green>
                <blue>118</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>117</green>
                <blue>117</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>97</red>
                <green>117</green>
                <blue>118</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>117</green>
                <blue>117</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>240</red>
                <green>240</green>
                <blue>240</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>51</red>
                <green>153</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="RoundProgressBar" name="roundBar3" native="true"/>
        </item>
        <item row="2" column="0">
         <widget class="RoundProgressBar" name="roundBar4" native="true">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Shadow">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>117</green>
                <blue>117</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Shadow">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>117</green>
                <blue>117</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Shadow">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>117</green>
                <blue>117</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="RoundProgressBar" name="roundBar5" native="true">
          <property name="palette">
           <palette>
            <active>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>117</green>
                <blue>117</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>24</red>
                <green>189</green>
                <blue>155</blue>
               </color>
              </brush>
             </colorrole>
            </active>
            <inactive>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>255</red>
                <green>117</green>
                <blue>117</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>24</red>
                <green>189</green>
                <blue>155</blue>
               </color>
              </brush>
             </colorrole>
            </inactive>
            <disabled>
             <colorrole role="Base">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>240</red>
                <green>240</green>
                <blue>240</blue>
               </color>
              </brush>
             </colorrole>
             <colorrole role="Highlight">
              <brush brushstyle="SolidPattern">
               <color alpha="255">
                <red>51</red>
                <green>153</green>
                <blue>255</blue>
               </color>
              </brush>
             </colorrole>
            </disabled>
           </palette>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="RoundProgressBar" name="roundBar6" native="true"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="base_circle_ckb">
          <property name="text">
           <string>外圆</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="data_circle_ckb">
          <property name="text">
           <string>数据圆</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="center_circle_ckb">
          <property name="text">
           <string>内圆</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="text_ckb">
          <property name="text">
           <string>文字</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QSlider" name="valueSlider">
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>30</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBothSides</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TaiChiWait</class>
   <extends>QWidget</extends>
   <header>taichiwait.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CircleWait</class>
   <extends>QWidget</extends>
   <header>circlewait.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PieWait</class>
   <extends>QWidget</extends>
   <header>piewait.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineWait</class>
   <extends>QWidget</extends>
   <header>linewait.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DonutWait</class>
   <extends>QWidget</extends>
   <header>donutwait.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ZoomCircleWait</class>
   <extends>QWidget</extends>
   <header>zoomcirclewait.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RoundProgressBar</class>
   <extends>QWidget</extends>
   <header>roundprogressbar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
